gpu_bdb/queries/q08/gpu_bdb_query_08_dask_sql.py

```diff
@@ -106,9 +114,11 @@ def main(data_dir, client, c, config):
     unique_review_sales = sessionized.map_partitions(
         get_unique_sales_keys_from_sessions, review_cat_code=REVIEW_CAT_CODE
     )
-    
-    unique_review_sales = unique_review_sales.to_frame()
-
+    if isinstance(merged_df, dask_cudf.DataFrame):
+        unique_review_sales = unique_review_sales.to_frame()
+    else: 
+        unique_review_sales = dd.from_dask_array(unique_review_sales, columns='wcs_sales_sk').to_frame() 
```

Comment:
Whats `unique_review_sales ` when using the pandas backend ?

Reply:
This is `unique_review_sales` output:
 ```
     wcs_sales_sk
0            12214
1            12985
2            20755
3            12669
4            18811
...            ...
8642         23135
8643         19731
8644         27443
8645         21328
8646         14237
```

Reply:
Sorry should have been clearer, i meant the type of `unique_review_sales` . 
```python3
>> type(unique_review_sales) 
```

Reply:
It's a `Dask.array` when using the Pandas backend and a `dask_cudf.Series` when using the cuDF backend

Reply:
Thanks for the clarification, i guess its because `.unique` returns `cudf.Series` for `cuDF` and `np.array` for `pandas`. 

Opened an issue to get this fixed upstream.  

 https://github.com/rapidsai/cudf/issues/10801

We might need to remove this branching when above is fixed.